client/app/resources/i18n/domain/Messages_fr_FR.xml

```diff
@@ -586,7 +586,8 @@
    <translation id="domain_configuration_dns_entry_add_publickey_revoke" qtlid="192396">Révoquer la clé publique</translation>
    <translation id="domain_configuration_dns_entry_add_servicetypes" qtlid="192409">Types de service</translation>
    <translation id="domain_configuration_dns_entry_add_servicetypes_all" qtlid="69173">Tous</translation>
-   <translation id="domain_configuration_dns_entry_add_servicetypes_email" qtlid="192422">e-mail</translation>
+   <translation id="domain_configuration_dns_entry_add_servicetypes_email" qtlid="192422">E-mail</translation>
+   <translation id="domain_configuration_dns_entry_add_servicetypes_none" qtlid="30646">Aucun</translation>
```

Comment:
I'm sorry but i don't notice that before, but, you should remove `qtlid`attribute for new translations ... I will update the CONTRIBUTING file for this, it's not specified ;)